defender-endpoint/mde-plugin-wsl.md

```diff
@@ -293,7 +296,30 @@ DeviceProcessEvents
          dnsProxy=false
          ```
 
-   - If the connectivity issues persist, collect the networking logs using the method mentioned at [link](https://aka.ms/wsllogs), and include those logs with your support bundle.
+   - If the connectivity issues persist, run the following steps to collect the networking logs
+
+      1. Open an elevated(admin) powerhsell prompt 
+      
+      2. Download and Run: `.\collect-networking-logs.ps1`
+
+         ```powershell
+         Invoke-WebRequest -UseBasicParsing "https://raw.githubusercontent.com/microsoft/WSL/master/diagnostics/collect-networking-logs.ps1" -OutFile collect-networking-logs.ps1
+         Set-ExecutionPolicy Bypass -Scope Process -Force
+         .\collect-networking-logs.ps1
+         ```
+
+      3.	Open a new command prompt and run: `wsl`
+      
+      4.	Open an elevated(admin) command prompt and run:  `wsl --debug-shell`
+
+      5.	In debug shell, run: `mdatp connectivity test`
+
+      6.	Allow the connectivity test to be completed
+
+      7.	Stop the .ps1 ran in step #2
+
+      8.	Share the generated .zip file along with support bundle.
```

Comment:
Mention cmd to collect support bundle in a separate step or link to existing doc on how to do it.